dashboard/config/linux/stable-5.15-arm64-kasan.config

```diff
@@ -9993,7 +9993,10 @@ CONFIG_CIFS_SWN_UPCALL=y
 CONFIG_CIFS_SMB_DIRECT=y
 CONFIG_CIFS_FSCACHE=y
 # CONFIG_CIFS_ROOT is not set
-# CONFIG_SMB_SERVER is not set
+CONFIG_SMB_SERVER=y
+# CONFIG_SMB_SERVER_SMBDIRECT is not set
```

Comment:
What do these 3 disabled configs do? Do we need to enable them to get more coverage?

Reply:
IDK. I wanted to get at least some coverage first.
We can discuss the tuning with @yskelg.

Reply:
@dvyukov are you typically trying to enable as much as possible for every opted-in subsystem?

Reply:
Let's discuss with @yskelg then.
The problem with such things is that if one does a change like this, it's likely nobody will ever get back to this later and refine/fix. So I usually try to do at least minimal homework researching around.
Think of it this way: with this change you have just defined the Linux kernel SMB fuzzing strategy for the next decade :)